### PR TITLE
Did I follow JS best practices?

### PR DESCRIPTION
**Pull request by zemola**
### Did I follow JS best practices?

This pull request is to Check the quality of the code having in mind the following:
JavaScript best practices.
DRY, KISS, and YAGNI rules.
HTML & CSS best practices.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>To-do list</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <main>
+      <!-- Todo list header -->
+      <div class="general-wrapper">
+      <div class="title">
+        <h1 id="title-id">To-Do list</h1>
+        <span class="material-icons">autorenew</span>
```
Comment (roshan-bajgain) [inline dist/index.html:19]: It would be better to use header in place of main section because this the heading part.

**Comment by roshan-bajgain** [comment]
Hi @zemola Everything is fine you are ahead of your project but It would be better if you change the edit and delete buttons because they are appearing at the same time and the requirement is that the user clicks the edit button then the delete button should appear with the background color.
Cheers and Happy coding

**Comment by zemola** [comment]
> Hi @zemola Everything is fine you are ahead of your project but It would be better if you change the edit and delete buttons because they are appearing at the same time and the requirement is that the user clicks the edit button then the delete button should appear with the background color. Cheers and Happy coding

**Comment by zemola** [comment]
Thank you, will look into it and get back to you.

**Pull request merged**